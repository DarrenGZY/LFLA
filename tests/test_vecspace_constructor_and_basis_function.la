function main()
{   
    vector v1 = [1,2,3];
    vector v2 = [2,3,4];
	vector v3 = [3,5,7];
	vecspace vs1 = L(v1, v2);
	vecspace vs2 = L(v1, v2, v3);
	vecspace vs3 = L();
	vector vecs[2] = {v1,v2,v3};
	vecspace vs4 = L(vecs);
    print(basis(vs1));
    print(basis(vs2));
	print(basis(vs3));
	print(basis(vs4));
}
