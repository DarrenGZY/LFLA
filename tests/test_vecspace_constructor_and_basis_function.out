[array([1, 2, 3]), array([2, 3, 4])]
[array([1, 2, 3]), array([2, 3, 4])]
[]
[array([1, 2, 3]), array([2, 3, 4])]
